src/helpers/convertToTypesFromSchemaProperties.ts

```diff
@@ -0,0 +1,89 @@
+import { getSchemaProperties } from '../shared';
+import { SwaggerProps, SwaggerSchema } from '../types';
+import { getDescription } from './getDescription';
+import { getResultStringForAdditionalPropertiesType } from './getResultStringForAdditionalPropertiesType';
+import { getResultStringForArrayType } from './getResultStringForArrayType';
+import { getResultStringForArrayWithoutItemRef } from './getResultStringForArrayWithoutItemRef';
+import { getResultStringForBooleanType } from './getResultStringForBooleanType';
+import { getResultStringForDictionaryKey } from './getResultStringForDictionaryKey';
+import { getResultStringForNumberType } from './getResultStringForNumberType';
+import { getResultStringForStringType } from './getResultStringForStringType';
+import { getResultStringType } from './getResultStringType';
+import { parseFormat } from './parseFormat';
+import { parseProperty } from './parseProperty';
+import { parseRefType } from './parseRefType';
+
+export const convertToTypesFromSchemaProperties = ({
```

Comment:
Originally from `typesConverter`.
Main change is to use `getResultStringType` helper to determine what kind of result string we have to write.
And then, have helpers for each scenarios instead of having it inline

Reply:
for the others helpers, if I don't comment on it, it's because the helpers was simply moved from `typesConverter` without change - except for typing